fe/fe-core/src/main/java/com/starrocks/sql/parser/AstBuilder.java

```diff
@@ -5183,6 +5180,11 @@ public ParseNode visitCreateFunctionStatement(StarRocksParser.CreateFunctionStat
                 properties.put(property.getKey(), property.getValue());
             }
         }
+        String inlineContent = null;
+        if (context.content != null) {
+            String text = context.content.getText();
+            inlineContent = text.substring(5, text.length() - 3);
```

Comment:
This is easy to obtain a unstandard format content. For example:
```
BEGIN  def a(b):
retrun b END
```
Should we check it here?

Reply:
we will forward it to python interperter directly.

Reply:
![img_v3_02b5_fe2ca3cb-280c-4df8-b812-d092ff3dc46g](https://github.com/StarRocks/starrocks/assets/34912776/11735df9-aa5b-4c9f-9033-db6951c3e199)

Reply:
Ok, please use `"BEGIN".length` to replcae 5